### ISSUE TITLE
Parenthesis get removed

### ISSUE THREAD

**Issue by kobyfullerton** [prettier/prettier#6100]
Hello everyone,

this is my first post on Github, so if I screwed up something or this was already mentioned somewhere my apologies. (still getting to know the ins and outs of the search function...) I just started learning JS, working on some school examples and was using Prettier to format the code and have noticed one quirk - when I format the code, the parenthesis get removed and thus the calculation for the amount is not correct. Please see the example below. 

Kind regards,
Koby

**Prettier 1.9.0**

**Input:**
function calculateFinalPurchaseAmount(amt) {
  amt = amt + (amt * TAX);
  return amt;
}

**Output:**
function calculateFinalPurchaseAmount(amt) {
  amt = amt + amt * TAX;
  return amt;
}

**Expected behaviour:**
function calculateFinalPurchaseAmount(amt) {
  amt = amt + (amt * TAX);
  return amt;
}

**Comment by lydell** [comment]
Hi!

JavaScript uses evaluates `+`, `-`, `*` and `/` in the same order as in maths, so `amt + amt * TAX` means the same thing as `amt + (amt * TAX)`. This is called [operator precedence](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence).

You can easily test this in the JavaScript console:

```
> 1 + 2 * 3
7
> 1 + (2 * 3)
7
> (1 + 2) * 3
9
```

Prettier does not print unnecessary parentheses for common math operations – that’s why they are “removed” after you run Prettier. So your bug must be hiding somewhere else!

See also #187.

**Comment by lydell** [comment]
Also, you seem to be running a very old version of Prettier – 1.9.0. The latest as of this writing is 1.17.0. I’d suggest updating to get a lot of fixes and improvements!

**Comment by kobyfullerton** [comment]
Hello Lydell,

thank you for your comment. With regards to math operator precedence, you are absolutely right - I am aware of the rules and I still don't know what I was looking at. The numbers that came out as a result, were not correct and of course, this was the first obvious thing I saw that changed, forgetting completely that it doesn't matter in which order the formula was written. 
When I finally came to the bottom of this and realised that the result I was getting was the string and not the numbers...  Sorry for the false alarm.
Regarding the version, I was looking at the changelog tab on the Mac version: 
[Unreleased]
[1.9.0]
Prettier 1.17
